tasks/libs/ciproviders/github_api.py

```diff
@@ -1,7 +1,6 @@
 import base64
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
`tasks/libs/ciproviders/*` ownership should be `DataDog/agent-ci-experience`, will do the change after this PR